tests/acl/test_stress_acl.py

```diff
@@ -10,79 +11,103 @@
 logger = logging.getLogger(__name__)
 
 LOOP_TIMES_LEVEL_MAP = {
-    'debug': 10,
-    'basic': 1000,
-    'confident': 10000
+    'debug': 100,
+    'basic': 500,
+    'confident': 5000
 }
 
 # Template json file used to test scale rules
-IP_ACL_FILE = "/tmp/acltb_test_stress_acl.json"
-IP_ACL_BASH_TEMPLATE = "acl/templates/acltb_test_stress_acl.sh"
-IP_ACL_BASH_FILE = "/tmp/acltb_test_stress_acl.sh"
-IP_ACL_BASH_LOG_FILE = "/tmp/acltb_test_stress_acl.log"
-
-
-def generate_acl_rule(duthost, ip_type):
-    """
-    Generate acl rule with template json file
-    """
-    rules_data = {}
-    ip_acl_entry = {}
-
-    ip_type == "ipv4"
-    acl_entry = {}
-    acl_entry[1] = {
-                    "actions": {
-                        "config": {
-                            "forwarding-action": "ACCEPT"
-                        }
-                    },
-                    "config": {
-                        "sequence-id": 1
-                    },
-                    "ip": {
-                        "config": {
-                            "source-ip-address": "20.0.0.1/32"
-                        }
-                    }
-                }
-    ip_acl_entry.update(acl_entry)
-    rules_data['acl'] = {
-        "acl-sets": {
-            "acl-set": {
-                "IP_STRESS_ACL": {
-                    "acl-entries": {
-                        "acl-entry": ip_acl_entry
-                    }
-                }
-            }
-        }
-    }
-
-    duthost.copy(content=json.dumps(rules_data, indent=4), dest=IP_ACL_FILE)
-
-
-def test_acl_add_del_stress(duthosts, rand_one_dut_hostname, get_function_conpleteness_level):
-    duthost = duthosts[rand_one_dut_hostname]
-    generate_acl_rule(duthost, "ipv4")
-    duthost.shell("config acl add table -p PortChannel101,PortChannel102,PortChannel103,PortChannel104 \
-                  IP_STRESS_ACL L3")
+STRESS_ACL_TABLE_TEMPLATE = "acl/templates/acltb_test_stress_acl_table.j2"
+STRESS_ACL_RULE_TEMPLATE = "acl/templates/acltb_test_stress_acl_rules.j2"
+STRESS_ACL_BASH_TEMPLATE = "acl/templates/acltb_test_stress_acl.sh"
+STRESS_ACL_TABLE_JSON_FILE = "/tmp/acltb_test_stress_acl_table.json"
+STRESS_ACL_RULE_JSON_FILE = "/tmp/acltb_test_stress_acl_rules.json"
+STRESS_ACL_BASH_FILE = "/tmp/acltb_test_stress_acl.sh"
+STRESS_ACL_BASH_LOG_FILE = "/tmp/acltb_test_stress_acl.log"
+
+LOG_EXPECT_ACL_TABLE_CREATE_RE = ".*Created ACL table.*"
+LOG_EXPECT_ACL_RULE_FAILED_RE = ".*Failed to create ACL rule.*"
+
+
+@pytest.fixture(scope='module')
+def setup_stress_acl_table(rand_selected_dut, tbinfo):
+    mg_facts = rand_selected_dut.get_extended_minigraph_facts(tbinfo)
+    pc = list(mg_facts['minigraph_portchannels'].keys())[0]
```

Comment:
TC may be failed on the T1 topology due to absent LAG ports on the DUT. Link to issue - https://github.com/sonic-net/sonic-mgmt/issues/6963. Could you please add extra verification of availability LAG ports on the DUT? For example:
```
    if not pc:
        pytest.skip('No portchannels found')
```